client/nats.go

```diff
@@ -151,60 +36,60 @@ func (n *natsClient) connect(bCtx *env.BubblyContext) error {
 		}))
 
 	if err != nil {
-		return fmt.Errorf(
+		return nil, fmt.Errorf(
 			`client failed to establish a connection to the NATS
-			server at address "%s": %w`,
-			n.Config.Addr,
+				server at address "%s": %w`,
```

Comment:
Is this a literal line break _inside_ the error message string literal? I wouldn't, if so 🙃